custom_components/google_home/translations/en.json

```diff
@@ -6,13 +6,15 @@
         "description": "If you need help with the configuration have a look here: https://github.com/leikoilja/ha-google-home. Use your google account username and app password. It's safer/easier to generate an app password and use it instead of the actual password. It still has the same access as the regular password, but still better than using the real password while scripting. (https://myaccount.google.com/apppasswords). If not, regular google account password should work.",
         "data": {
           "username": "Google account username",
-          "password": "Google account app password"
+          "password": "Google account app password",
+          "master_token": "Google account master token"
         }
       }
     },
     "error": {
       "auth": "Username/Password is incorrect. If your google account has 2FA enabled please generate app password (https://myaccount.google.com/apppasswords). If authentication still fails refer to https://github.com/leikoilja/ha-google-home#troubleshooting",
-      "pass-len": "Password length is limited to 100 characters by Google Policy (https://support.google.com/a/answer/139399). Please, change your password or use an app password (https://myaccount.google.com/apppasswords)."
+      "pass-len": "Password length is limited to 100 characters by Google Policy (https://support.google.com/a/answer/139399). Please, change your password or use an app password (https://myaccount.google.com/apppasswords).",
+      "missing-inputs": "Please provide either username and password or master token."
```

Comment:
since it's a new translation string not present in any other translation files, we might want to google translate it in all other integration supported languages

Reply:
Agreed. I'll add them